.github/workflows/tag.yml

```diff
@@ -0,0 +1,24 @@
+name: gocardless/atlantis/tag
+on:
+  push:
+    branches:
+    - master
+jobs:
+  docker_tag:
+    if: ${{ !contains('refs/heads//.*/', github.ref) }}
+    runs-on: ubuntu-latest
+    container:
+      image: golang:1.14
+      env:
+        GOFLAGS: "-mod=vendor"
+    steps:
+    - uses: actions/checkout@v2
+    - run: make build-service
+#     # 'setup_remote_docker' was not transformed because there is no suitable equivalent in GitHub Actions
```

Comment:
We'll probably have to install docker and login using something like [docker/login-action](https://github.com/docker/login-action#google-container-registry-gcr). This might mean using workload identity federation, see [our docs](https://backstage.gocardless.io/docs/default/component/github-actions/CI/03-authentication/#authenticating-with-workload-identity-federation) for further info here.